clients/client/src/test/java/org/projectnessie/model/TestModelObjectsSerialization.java

```diff
@@ -0,0 +1,202 @@
+            ImmutableTag.builder().hash(HASH).name("tagname").build(),
+            Tag.class,
+            "{\"type\":\"TAG\",\"name\":\"tagname\",\"hash\":\"" + HASH + "\"}"),
+        new Case(
+            ImmutableHash.builder().name(HASH).build(),
+            Hash.class,
+            "{\"type\":\"HASH\",\"name\":\"" + HASH + "\",\"hash\":\"" + HASH + "\"}"),
+        new Case(
+            ImmutableTransplant.builder()
+                .addHashesToTransplant(HASH)
+                .fromRefName(branchName)
+                .build(),
+            Transplant.class,
+            "{\"hashesToTransplant\":" + "[\"" + HASH + "\"]," + "\"fromRefName\":\"testBranch\"}"),
+        new Case(
+            ImmutableEntriesResponse.builder()
+                .addEntries(
+                    ImmutableEntry.builder()
+                        .type(Type.ICEBERG_TABLE)
+                        .name(ContentsKey.fromPathString("/tmp/testpath"))
+                        .build())
+                .token(HASH)
+                .hasMore(true)
+                .build(),
+            EntriesResponse.class,
+            "{\"hasMore\":true,"
+                + "\"token\":\""
+                + HASH
+                + "\","
+                + "\"entries\":[{\"type\":\"ICEBERG_TABLE\","
+                + "\"name\":{\"elements\":[\"/tmp/testpath\"]}}]}"),
+        new Case(
+            ImmutableLogResponse.builder()
+                .addOperations()
+                .token(HASH)
+                .addOperations(
+                    ImmutableCommitMeta.builder()
+                        .commitTime(now)
+                        .author("author@testnessie.com")
+                        .committer("committer@testnessie.com")
+                        .authorTime(now)
+                        .hash(HASH)
+                        .message("test commit")
+                        .putProperties("prop1", "val1")
+                        .signedOffBy("signer@testnessie.com")
+                        .build())
+                .hasMore(true)
+                .build(),
+            LogResponse.class,
+            "{\"hasMore\":true,"
+                + "\"token\":\""
+                + HASH
+                + "\","
+                + "\"operations\":["
+                + "{\"hash\":\""
+                + HASH
+                + "\","
+                + "\"committer\":\"committer@testnessie.com\",\"author\":\"author@testnessie.com\","
+                + "\"signedOffBy\":\"signer@testnessie.com\",\"message\":\"test commit\","
+                + "\"commitTime\":\""
+                + now
+                + "\","
+                + "\"authorTime\":\""
+                + now
+                + "\","
+                + "\"properties\":{\"prop1\":\"val1\"}}]}"),
+        new Case(
+            ImmutableMerge.builder().fromHash(HASH).fromRefName(branchName).build(),
+            Merge.class,
+            "{\"fromHash\":\"" + HASH + "\"," + "\"fromRefName\":\"testBranch\"}"));
```

Comment:
no need for `" + "` if its all on a single line (and in all the other places)

Reply:
I thought of using variable here because the full hash value looked polluting; less appealing on the readability.
Single string is definitely preferable from perf standpoint; but for tests, I thought of preferring readability. Let me know your thoughts.

Reply:
Not "complaining" about using `HASH`, but the other string concatenations.